src/tools/data/EXPORT_TABLES.ts

```diff
@@ -4,6 +4,10 @@ import { CycleName } from 'meta/assessment/cycle'
 import { Schemas } from 'server/db/schemas'
 import { ExportTableProps } from 'server/service/databaseService/export'
 
+export type ExportTableConfig = ExportTableProps & {
+  skipExport?: boolean
```

Comment:
why do we want to still use the skipExport ?

Reply:
I explained this yesterday in the meeting:

- We are using skipExport to skip the tables we have static fixtures for

Reply:
The subset of tables we discussed about is created here in `tools` which is passed to DB service:

<img width="975" height="389" alt="image" src="https://github.com/user-attachments/assets/5db14ec3-a91a-4054-ae0e-17fe3fa94880" />

Correct me if I'm wrong - what are you looking for exactly?